src/lib.rs

```diff
@@ -33,9 +62,9 @@ macro_rules! gcd_impl {
                     let r = a % b;
                     a = b;
                     b = r;
-                }
+		}
 
-                a
+		a
```

Comment:
You have tabs inserted here

Reply:
should `cargo fmt` have fixed that? Or is it because its inside a macro it doesn't work?

Reply:
IIRC `cargo fmt` does not touch macros at all